### PR TITLE
Required changes based on the new dotnet-docker repo

### PR DESCRIPTION
**Pull request by mirsaeedi**
Since with the release of dotnet 2.1, Microsoft decided to move its docker samples to a new repo, some changes are required to reflect this new decision.

1. links
2. versions (dotnet, docker, docker images)
3. instructions

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/core/docker/building-net-docker-images.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ To build and run, install the following items:
 
 #### Installing Docker Client
 
-Install [Docker 17.06](https://docs.docker.com/release-notes/docker-ce/) or later of the Docker client.
+Install [Docker 18.03](https://docs.docker.com/release-notes/docker-ce/) or later of the Docker client.
```
Comment (JRAlexander) [inline docs/core/docker/building-net-docker-images.md]: @richlander - Is this change ok?

File: docs/core/docker/building-net-docker-images.md [hunk 2]
```diff
@@ -131,10 +131,10 @@ The Docker client can be installed in:
...
 
-The easiest way to get the sample is by cloning the [samples repository](https://github.com/dotnet/dotnet-docker-samples) with git, using the following instructions: 
+The easiest way to get the sample is by cloning the [samples repository](https://github.com/dotnet/dotnet-docker) with git, using the following instructions: 
 
 ```console
-git clone https://github.com/dotnet/dotnet-docker-samples/
+git clone https://github.com/dotnet/dotnet-docker
 ```
 
 You can also download the repository (it is small) as a zip from the .NET Core Docker samples repository.
```
Comment (JRAlexander) [inline docs/core/docker/building-net-docker-images.md]: Should be ".NET Core Docker repository."

File: docs/core/docker/building-net-docker-images.md [hunk 3]
```diff
@@ -131,10 +131,10 @@ The Docker client can be installed in:
 
 ### Getting the sample application
 
-The easiest way to get the sample is by cloning the [samples repository](https://github.com/dotnet/dotnet-docker-samples) with git, using the following instructions: 
+The easiest way to get the sample is by cloning the [samples repository](https://github.com/dotnet/dotnet-docker) with git, using the following instructions:
```
Comment (JRAlexander) [inline docs/core/docker/building-net-docker-images.md]: Please change to ".NET Core Docker repository."

File: docs/core/docker/building-net-docker-images.md (hunk 2 above)
Comment (JRAlexander) [inline docs/core/docker/building-net-docker-images.md]: Please change to ".NET Core Docker repository."

**Review by JRAlexander — CHANGES_REQUESTED** [review]
Looks great, @mirsaeedi! Just two small things.

File: docs/core/docker/building-net-docker-images.md (hunk 1 above)
Comment (JRAlexander) [reply docs/core/docker/building-net-docker-images.md]: Yes, it's ok

File: docs/core/docker/building-net-docker-images.md (hunk 2 above)
Comment (mirsaeedi) [reply docs/core/docker/building-net-docker-images.md]: @JRAlexander  Thanks for your thorough review. I applied the requested change.

File: docs/core/docker/building-net-docker-images.md (hunk 3 above)
Comment (mirsaeedi) [reply docs/core/docker/building-net-docker-images.md]: @JRAlexander 👍 Done.

**Review by JRAlexander — APPROVED** [review]

**Comment by JRAlexander** [comment]
Thanks, @mirsaeedi! I just approved this and will merge straightaway. You should see it live in the next few days.

**Pull request merged**